### ISSUE TITLE
Failing to install 1.8

### ISSUE THREAD

**Issue by spalger** [shyiko/jabba#19]
When I run `jabba install 1.8` I'm seeing:

```
Downloading 1.8.112 (http://download.oracle.com/otn-pub/java/jdk/8u102-b15/jdk-8u112-linux-x64.tar.gz)
7073/7073
Extracting /tmp/jabba-d-494731954 to /home/ubuntu/.jabba/jdk/1.8.112

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

'tar xzf /tmp/jabba-d-494731954 --strip-components=1 -C /home/ubuntu/.jabba/jdk/1.8.112' failed: exit status 2
```

When I visit that URL I see:

![image](https://cloud.githubusercontent.com/assets/1329312/19679323/33ca0736-9a56-11e6-9d98-e7192642745c.png)

**Comment by shyiko** [comment]
@spalger Thank you for letting me know. Fixed in 5c687bcf8ee8280c0748a5ea1711683f48e8f4a8. 
I'm gonna make sure situation like this doesn't happen again via https://github.com/shyiko/jabba/issues/20.

🙇

**Comment by spalger** [comment]
no thank you!
